Project6b/Project6b/ViewController.swift

```diff
@@ -61,8 +61,17 @@ class ViewController: UIViewController {
         var previous: UILabel?
         
         for label in [label1, label2, label3, label4, label5] {
-            label.widthAnchor.constraint(equalTo: view.widthAnchor).isActive = true
-            label.heightAnchor.constraint(equalToConstant: 88).isActive = true
+            //1. leadingAnchor and trailingAnchor
+//            label.leadingAnchor.constraint(equalTo: view.leadingAnchor).isActive = true
+//            label.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
+            //2. using the safeAreaLayoutGuide
+            label.safeAreaLayoutGuide.leadingAnchor.constraint(equalTo: view.leadingAnchor).isActive = true
+            label.safeAreaLayoutGuide.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
+            //3. making the height of your labels equal to 1/5th
+            label.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 1/5, constant: -10).isActive = true
```

Comment:
W tasku było proszone o użycie `safeAreaLayoutGuide` - 
Różnica pomiędzy samym view a view.safeLayoutGuide jest taka, że view to od górnej krawędzi do dolnej ekranu, a safe layout guide pomija obszar na górze i dole: 
![image](https://github.com/user-attachments/assets/02bdef52-2a23-4510-beb3-3033989dcdad)

Zamiast 1/5 dałbym 0.2  - 1/5 zadziała, ale 0.2 wygląda spójniej :D